examples/flutter_counter/test/counter/view/counter_view_test.dart

```diff
@@ -61,7 +61,7 @@ void main() {
       final decrementFinder = find.byKey(_decrementButtonKey);
       await tester.ensureVisible(decrementFinder);
       await tester.tap(decrementFinder);
-      verify(counterCubit.decrement()).called(1);
+      verify(counterCubit).called(#decrement).once();
```

Comment:
Out of curiosity, but I want to make sure we cover all cases: if we use symbols, how can we verify a call with arguments?

For example, if the signature of decrement were `void decrement(int value)`, would you still do 👇 ?

```dart
verify(counterCubit).called(#decrement).once();
```

Reply:
We can do
```dart
verify(counterCubit).called(#decrement).withArgs(positional: [any]).once();

```